src/cards/ResourceCard/style.js

```diff
@@ -10,6 +10,10 @@ const useStyles = makeStyles(() => ({
     flexDirection: 'column',
     justifyContent: 'space-between',
   },
+  description: {
+    height: '70px',
+    overflow: 'auto',
+  },
```

Comment:
Is it not possible to take the opportunity to delete `style.js` and implement style directly inside `sx` balise ?

Reply:
IIRC the `sx` prop can only be used on _mui_ components, whereas we want to use it on a `div` here